enumerable.rb

```diff
@@ -0,0 +1,88 @@
+module Enumerable
+  def my_each
+    count = 0
+    while count < self.size
+      yield(self[count])
+      count += 1
+    end
+
+  def my_each_width_index
+    index = 0

```

Comment:
`my_each_width_element` doesn't work
You are trying to yield `self[element]` which is not an index. You can just `yield(index, element)`

Reply:
done